x-pack/plugins/ml/public/application/components/data_grid/feature_importance/use_classification_path_data.tsx

```diff
@@ -49,30 +56,43 @@ export const getStringBasedClassName = (v: string | boolean | undefined | number
   return v;
 };
 
+export const formatValue = (number: number, precision = 3, fractionDigits = 1): string => {
+  if (Math.abs(number) < 10) {
+    return Number(number.toPrecision(precision)).toString();
+  }
+  return number.toFixed(fractionDigits);
+};
+
 export const useDecisionPathData = ({
   baseline,
   featureImportance,
   predictedValue,
 }: UseDecisionPathDataParams): { decisionPathData: DecisionPathPlotData | undefined } => {
   const decisionPathData = useMemo(() => {
-    return baseline
-      ? buildRegressionDecisionPathData({
-          baseline,
+    if (baseline !== undefined) {
```

Comment:
nit: to prevent an extra indentation, use 
```suggestion
    if (baseline === undefined) return;
```

Reply:
Updated here 10eae2d